cmd/bnscli/cmd_gov.go

```diff
@@ -471,3 +471,43 @@ Reads a transaction from the input and attaches the provided elector address, we
 	_, err = writeTx(output, tx)
 	return nil
 }
+
+func cmdUpdateElectionRule(input io.Reader, output io.Writer, args []string) error {
+	fl := flag.NewFlagSet("", flag.ExitOnError)
+	fl.Usage = func() {
+		fmt.Fprintln(flag.CommandLine.Output(), `
+Creates a new version for an existing election rule. The new version is used for new proposals.
+		`)
+		fl.PrintDefaults()
+	}
+	var (
+		id            = flSeq(fl, "id", "", "The ID of the election rule")
+		durationFl    = fl.Int("voting-period", 0, "Duration in seconds how long the voting period will take place")
+		numeratorFl   = fl.Int("threshold-numerator", 0, "The top number of the fraction.")
+		denominatorFl = fl.Uint("threshold-denominator", 0, "The bottom number of the fraction")
```

Comment:
How about requesting a human readable format. I think it is easier to understand something like `-threshold 2/3` instead of `-threshold-numerator 2 -threshold-denominator 3`. Parsing human readable format should be quite easy.

Reply:
In fact I had the same idea here, too but 2 arguments are simpler to handle than 1. But I agree `2/3` would be a better UX. unless something odd happens on the shell